OntologyMapper.Mapped/OntologyMapper.Mapped.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This either needs its own independent pipeline build or incorporated with the exiting one.